src/legacy/core_plugins/kibana/public/management/sections/objects/lib/__jest__/get_relationships.test.ts

```diff
@@ -21,20 +21,20 @@ import { getRelationships } from '../get_relationships';
 
 describe('getRelationships', () => {
   it('should make an http request', async () => {
-    const $http = jest.fn();
+    const $http = jest.fn() as any;
```

Comment:
$http a callable class. Too difficult to properly type even with jest utilities. This is going to be removed anyway when migrating to NP apis

Reply:
Is this file going to be removed?

Reply:
not the file, but the `$http` service is going to be replaced by core's `http` service, which provide proper mocks.

Reply:
I'm asking because in the NP we keep test files side by side to the code, while here it's under a separate `__jest__` folder.

Reply:
Wasn't sure about the jest test-files pattern for legacy, but it seems I can safely move them one level up. Will do.